app/controllers/support_interface/provider_users_controller.rb

```diff
@@ -26,10 +38,13 @@ def edit
 
     def update
       provider_user = ProviderUser.find(params[:id])
+      @form = ProviderUserForm.from_provider_user(provider_user)
 
-      @form = ProviderUserForm.new(provider_user_params.merge(provider_user: provider_user))
+      @form.first_name = provider_user_params[:first_name]
+      @form.last_name = provider_user_params[:last_name]
+      @form.provider_ids = provider_user_params[:provider_ids]
```

Comment:
I think we get `#assign_attributes` along with `ActiveModel::Model`, which might make this a bit more succinct?